### PR TITLE
Add sklearn code snippet

### PR DESCRIPTION
**Pull request by osanseviero**
Screen Shot 2021-06-24 at 8 41 02 AM

### CODE REVIEW

File: interfaces/Libraries.ts [hunk 1]
```diff
@@ -161,6 +162,32 @@ const timm = (model: ModelData) =>
...
+`from huggingface_hub import hf_hub_url, cached_download
+import joblib
+
+# No joblib file found!
+model = joblib.load(cached_download(
+	hf_hub_url("${model.modelId}", "name of file")
+))`;
+
+const sklearn = (model: ModelData) => {
+	const joblib_file = model.siblings?.find(file => file.rfilename.endsWith(".joblib"));
```
Comment (julien-c) [inline interfaces/Libraries.ts:183]: In my mind we were enforcing a model filename of `sklearn_model.joblib` (similar to what we're doing for pt/tf/jax/etc)

Is there any downside to enforcing a fixed filename?

**Comment by julien-c** [comment]
Also this screenshot makes me think that maybe we need/want a `hf_hub_download(...)` method which does `cached_download(hf_hub_url())`

cc @LysandreJik

**Comment by LysandreJik** [comment]
Yeah that sounds great! I'll open a PR in a bit.

File: interfaces/Libraries.ts (hunk 1 above)
Comment (osanseviero) [reply interfaces/Libraries.ts:183]: No strong opinion on this, just trying to be a bit more flexible.

Let's go with the fixed filename as we do for pt/tf/jax :)

Comment (julien-c) [reply interfaces/Libraries.ts:183]: yeah otherwise someone will have multiple .joblib files in a repo and we don't know which one to run in inference :)

**Comment by osanseviero** [comment]
Friendly ping :)

**Comment by julien-c** [comment]
LGTM!

**Review by julien-c — APPROVED** [review]

**Pull request merged**